script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -29,23 +29,23 @@ All that's here is my own futile imagination, the sound of my heart pumping bloo
 <Page9>
 It was out of the blue.
 "Tohno Makihisa has passed away.
-Tohno Shiki is to return at once to the Tohno household."
-The telegram from my family home, which I hadn't heard from in nearly seven years, was something to that effect.
+Tohno Shiki is to return to the Tohno household at once."
+The telegram from my original home, which I hadn't heard from in nearly seven years, was something to that effect.
 <Page10>
 I can't go against a decision made by the main Tohno family. I am still a student, and I rely on them for living expenses.
-It was a little stubborn of me that I chose to leave the morning of the day on which I was to return, rather than the night before.
+Since I was being stubborn, I decided to leave the morning of the day I had to return, rather than the night before.
 <Page11>
-I just wanted to stay as long as possible in the home I had lived in until now...
+I just wanted to stay as long as possible in the home I have been living in until now...
 Even though it was my family's decision, it was my way of showing respect to the Arimas, who had welcomed me, a person that they had absolutely no connection with, into their family.
 <Page12>
-"Thank you for the food. Today's breakfast was delicious as always."
+"Thank you for the food. Today's breakfast was delicious, as always."
 Before the sun had risen I ate breakfast, left the table, and pressed my palms together in the room that had been mine.
 <Page13>
 It was a meager way to express my gratitude for all the years I had spent here, but if I left my feelings behind, I would leave feeling unfulfilled.
 Life here was nothing but bright. That was all the baggage I needed to take with me.
 <Page14>
 When I left the Arima household, Keiko-san was the only one to see me off.
-It was my own suggestion that I should be as quiet as possible so as not to wake the rest of the family.
+It was my own suggestion that I should be as quiet as possible not to wake the rest of the family.
```

Comment:
I'd revert this - you need a "so as" or "in order to" to conjoin the action and effect